src/js/load.js

```diff
@@ -110,7 +110,8 @@ class Application extends Component {
 
         controller.on('profile.loaded', payload => {
```

Comment:
can we move this into one of the js files in the `setup` folder

Reply:
Should i make a new file for this @emre2038 
If yes what should i name this ?

Reply:
Maybe a good place for this is https://github.com/OpenUpSA/wazimap-ng-ui/blob/staging/src/js/setup/general.js